src/infill.cpp

```diff
@@ -388,7 +388,8 @@ void Infill::generateCrossInfill(const SierpinskiFillProvider& cross_fill_provid
 
 void Infill::addLineSegmentsInfill(Polygons& result, Polygons& input)
 {
-    ClipperLib::PolyTree interior_segments_tree = in_outline.lineSegmentIntersection(input);
+	ClipperLib::PolyTree interior_segments_tree;
+	in_outline.lineSegmentIntersection(input, interior_segments_tree);
```

Comment:
Code style: Use spaces instead of tabs.

This is also where the merge conflict lies. You can fix the merge conflict and the tab indenting by replacing these lines with:

```
    ClipperLib::PolyTree interior_segments_tree;
    in_outline.offset(infill_overlap).lineSegmentIntersection(input, interior_segments_tree);
```

Reply:
Fixed in the merge commit here: https://github.com/Ultimaker/CuraEngine/commit/05e93dabce9e863b8742fd69ed87717e1594e7a9